### ISSUE TITLE
ASB failed to start 

### ISSUE THREAD

**Issue by gbenhaim** [openshift/ansible-service-broker#621]
After installing Openshift 3.7 using openshift-ansible I'm getting the following error:

```ESC[31m[2018-01-09T10:18:57.14Z] [ERROR] - Unable to retrieve cluster roles rules from cluster
 You must be using OpenShift 3.7 to use the User rules check.
the server could not find the requested resourceESC[0m
```
`OpenShift version: v3.7.0+7ed6862`
`Kubernetes version: v1.7.6+a08f5eeb62`

The full log of the ASB pod:

```
Using config file mounted to /etc/ansible-service-broker/config.yaml
============================================================
==           Starting Ansible Service Broker...           ==
============================================================
ESC[32m[2018-01-09T10:18:53.251Z] [NOTICE] - Initializing clients...ESC[0m
[2018-01-09T10:18:53.251Z] [INFO] - == ETCD CX ==ESC[0m
[2018-01-09T10:18:53.251Z] [INFO] - EtcdHost: asb-etcd.openshift-ansible-service-broker.svcESC[0m
[2018-01-09T10:18:53.251Z] [INFO] - EtcdPort: 2379ESC[0m
[2018-01-09T10:18:53.251Z] [INFO] - Endpoints: [https://asb-etcd.openshift-ansible-service-broker.svc:2379]ESC[0m
2018/01/09 10:18:53 Unable to get log.logfile from config
[2018-01-09T10:18:53.269Z] [INFO] - Etcd Version [Server: 3.2.13, Cluster: 3.2.0]ESC[0m
[2018-01-09T10:18:53.277Z] [INFO] - OpenShift version: v3.7.0+7ed6862ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Kubernetes version: v1.7.6+a08f5eeb62ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - == REGISTRY CX == ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Name: localregistryESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Type: local_openshiftESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Url: ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - == REGISTRY CX == ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Name: dhESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Type: dockerhubESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Url: ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - == REGISTRY CX == ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Name: dhESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Type: dockerhubESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Url: ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - == REGISTRY CX == ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Name: localregistryESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Type: local_openshiftESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Url: ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Initiating Recovery ProcessESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Listening for provision messagesESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Listening for deprovision messagesESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Listening for provision messagesESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - No jobs to recoverESC[0m
ESC[32m[2018-01-09T10:18:53.278Z] [NOTICE] - ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Broker configured to bootstrap on startupESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - Attempting bootstrap...ESC[0m
[2018-01-09T10:18:53.278Z] [INFO] - AnsibleBroker::BootstrapESC[0m
[2018-01-09T10:18:53.341Z] [INFO] - Validating specs...ESC[0m
ESC[32m[2018-01-09T10:18:53.341Z] [NOTICE] - All specs passed validation!ESC[0m
!!!!!!filter mode is none[2018-01-09T10:18:55.339Z] [INFO] - Validating specs...ESC[0m
ESC[32m[2018-01-09T10:18:55.339Z] [NOTICE] - All specs passed validation!ESC[0m
!!!!!!filter mode is none[2018-01-09T10:18:56.746Z] [INFO] - Validating specs...ESC[0m
ESC[32m[2018-01-09T10:18:56.746Z] [NOTICE] - All specs passed validation!ESC[0m
[2018-01-09T10:18:56.816Z] [INFO] - Validating specs...ESC[0m
ESC[32m[2018-01-09T10:18:56.816Z] [NOTICE] - All specs passed validation!ESC[0m
ESC[32m[2018-01-09T10:18:56.816Z] [NOTICE] - Broker successfully bootstrapped on startupESC[0m
ESC[31m[2018-01-09T10:18:57.14Z] [ERROR] - Unable to retrieve cluster roles rules from cluster
 You must be using OpenShift 3.7 to use the User rules check.
the server could not find the requested resourceESC[0m
```

**Comment by maleck13** [comment]
There was a bug with the start up, which has now been resolved. https://github.com/openshift/ansible-service-broker/pull/618
A work around for this currently is to set the config broker options ```auto_escalate``` to true and redeploy the broker

**Comment by gbenhaim** [comment]
Thanks.

**Comment by rthallisey** [comment]
closing this since it's fixed by #618. Thanks @maleck13 for the workaround.
